etcdutl/snapshot/v3_snapshot.go

```diff
@@ -471,8 +480,10 @@ func (s *v3Manager) saveWALAndSnap() (*raftpb.HardState, error) {
 		return nil, err
 	}
 
-	// add members again to persist them to the backend we create.
-	be := backend.NewDefaultBackend(s.lg, s.outDbPath())
+	// add members again to persist them to the store we create.
+	st := v2store.New(etcdserver.StoreClusterPrefix, etcdserver.StoreKeysPrefix)
+	s.cl.SetStore(st)
```

Comment:
can you explain why adding this (v2store)?

Reply:
Good catch. I'm reviewing the original PR, and I probably wrongly fixed a conflict, as the implementation of `v3_snapshot` had that `v2store` before, and it's not here anymore. I'll check for any other differences with the original PR and update this PR.

Reply:
Removed.